torch/csrc/jit/init.cpp

```diff
@@ -117,14 +118,17 @@ void initJITBindings(PyObject* module) {
           [](std::shared_ptr<Graph>& g) { return PropagateQuantInfo(g); })
       .def(
           "_jit_pass_insert_observers",
-          [](std::shared_ptr<Graph>& g, py::function pyObserverFunction) {
+          [](std::shared_ptr<script::Module> mod,
+             py::function pyObserverFunction) {
             // Create a new node that would be used in the insert observer pass:
             // all observer nodes will be cloned from this one.
-            Node* new_node = g->createPythonOp(
+            Graph g;
```

Comment:
> why not use graph contained in top level module method? Does it not make difference

Because we don't need this node to exist in any actual graph - we just use it to copy from and then delete. Creating a temporary graph for this seems 1) easier, 2) better aligned with the semantics.